Assets/Scripts/CaptureSceneManager.cs

```diff
@@ -174,6 +214,7 @@ public void CheckAnswer(int buttonId)
             correctStatusShadow.color = new Color32(100, 10, 10, 255);
             SetCorrectStatus("Incorrect");
         }
+        Destroy(captureViewBeastie.gameObject);

```

Comment:
Can we move this to go under the Update() method within delay statement or something similar? Example:

```
if (sceneTransitionDelay <= 0)
{
    Destroy(captureViewBeastie.gameObject);
    gameManager.SetAllBeastiesActive(true);
    SceneManager.LoadScene(Constant.OverworldMap);
}
```

This way, the "Correct"/"Incorrect" status result shows in the UI long enough (1.5 seconds) to be readable instead of flashing by.